src/site/markdown/server/manage-cli.md

```diff
@@ -258,6 +258,22 @@ Moreover, James allows to specify defaults maximum values, at the server level.
 {cli} GetDefaultMaxStorageQuota
 ```
 
+## Manage Sieve Scripts
+
+You can add and set a script as active:
```

Comment:
You can add a Sieve script for a user, and set it as active with the following command: